### DISCUSSION TITLE
搜出了奇怪的结果

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#143] [一般讨论]
结果的这串奇奇怪怪的是什么
![image](https://user-images.githubusercontent.com/66856838/107977831-c0aaf780-6ff6-11eb-8cc3-6941d4a279b1.png)
/moerens/851587c824128d1d5d91ef4089b21dcd
Author: /ch2/851587c824128d1d5d91ef4089b21dcd

**Comment by Tsuk1ko** [comment]
这你得问 ascii2d
